aci/resource_aci_fvepgselector_test.go

```diff
@@ -0,0 +1,150 @@
+package aci
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ciscoecosystem/aci-go-client/client"
+	"github.com/ciscoecosystem/aci-go-client/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccAciEndpointSecurityGroupEPgSelector_Basic(t *testing.T) {
+	var endpoint_security_group_e_pg_selector models.EndpointSecurityGroupEPgSelector
+	fv_tenant_name := acctest.RandString(5)
+	fv_ap_name := acctest.RandString(5)
+	fv_e_sg_name := acctest.RandString(5)
+	fv_e_pg_selector_name := acctest.RandString(5)
+	description := "endpoint_security_group_e_pg_selector created while acceptance testing"
```

Comment:
_e_pg_ -> _epg_

Reply:
Fix all the other occurence of this and of _e_sg_ -> _esg_

Reply:
Done